### ISSUE TITLE
Publish AWS common definitions

### ISSUE THREAD

**Issue by dpantke** [aws-cloudformation/cloudformation-cli#374]
I noticed that in your examples, for AWS resources ( which you are using internally I'd imagine ), you have a common definition file for types that can be included for reference. The one I'm specifically referring to here is aws.common.types.v1.json . I would like that to be published at a well-known URL so that I can reference it to get easy reference when dealing with common parameters I want validated like availability zone, region, ARN, etc....

**Comment by rjlohan** [comment]
These schemas are planned to be published soon - probably in the next couple of weeks. Stay tunes.

**Comment by tobywf** [comment]
I believe this is now available in some regions, for example: https://schema.cloudformation.us-east-1.amazonaws.com/aws.common.types.v1.json

**Comment by benkehoe** [comment]
Is there a root URL that makes these types discoverable?

**Comment by rjlohan** [comment]
> Is there a root URL that makes these types discoverable?

Yeah this should be announced soon, once it's available in all regions is usually the process.

**Comment by PatMyron** [comment]
[The GovCloud partition is incorrect and the isolated partitions are missing in the current `Arn` definition](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/#pkg-constants)

There will be more partitions in the future as well, so let's just generalize it beyond the current partitions

---

The `AvailabilityZone` definition also does not match [`us-west-2-lax-1a`](https://aws.amazon.com/blogs/aws/aws-now-available-from-a-local-zone-in-los-angeles/)
